libraries/AP_RCProtocol/AP_RCProtocol_Backend.cpp

```diff
@@ -21,6 +21,14 @@
 #include <AP_Vehicle/AP_Vehicle_Type.h>
 #include <AP_Logger/AP_Logger.h>
 
+// for video TX configuration:
+#if AP_VIDEOTX_ENABLED
```

Comment:
I don't think you can do this can you because the define is inside the video tx header

Reply:
I can get away with it because the header is transitively included via `AP_RCProtocol.h`

I've added an explicit include in this file.

.... that transitive include is a work of evil - the include at the end of AP_RCProtocol.h....